Files/MultilingualResources/Files.ru-RU.xlf

```diff
@@ -3051,7 +3051,7 @@
         </trans-unit>
         <trans-unit id="SettingsPinRecycleBin.Title" translate="yes" xml:space="preserve">
           <source>Pin Recycle Bin to favorites</source>
-          <target state="needs-review-translation">Закрепить Коризину на боковой панели</target>
+          <target state="needs-review-translation">Закрепить Корзину на боковой панели</target>
```

Comment:
Is this translation finished or do you want someone (or you in the future) to review it? If it's the first case, this should be:
```suggestion
          <target state="translated">Закрепить Корзину на боковой панели</target>
```

Apply this to the other lines (3116, 3128, 3148, 3168, )that have the same state (needs review instead translated).

Reply:
>Is this translation finished

Hello! I was not sure about the need to change the state. I fixed it.
>do you want someone (or you in the future) to review it?

I would like to add the authors of this localization or someone else for review, but i can't do it. For me, the translation of these strings is complete.

Reply:
@MarkBirkholz the "needs-review-translation" implies that that string is not guarantee to be accurate and needs someone to review in the future, while the "translated" means that string is accurate and there is no need to review it in the future (unless it changes, but this will make the string automatically change to the needs-review-translation state).

So, if you think your translation is accurate you can go with the translated state. If you are not sure, you can let the needs-review-translation so it's easy for others (or you) in the future to review those strings.

Let me know if you are okay letting those on the translated state or feel free to change it to needs-review-translation whatever string you think could be not accurate and needs to be reviewed.

Hope I explained well myself, if you have any questions feel free to ask since I'm not a well explainer 😆

Reply:
I found these typos while using the application and decided to fix them. I think this translation is ok and can be merged.